File: v131016/TeeNew/Exponential_Trend.pas
unit Exponential_Trend;
{$I TeeDefs.inc}

interface

uses
  {$IFNDEF LINUX}
  Windows, Messages,
  {$ENDIF}
  SysUtils, Classes,
  {$IFDEF CLX}
  QGraphics, QControls, QForms, QDialogs, QExtCtrls, QStdCtrls, QComCtrls,
  {$ELSE}
  Graphics, Controls, Forms, Dialogs, ExtCtrls, StdCtrls, ComCtrls,
  {$ENDIF}
  Base, CurvFitt, Series, TeEngine, TeeProcs, Chart;

type
  TExponentialTrendForm = class(TBaseForm)
    Series1: TFastLineSeries;
    TeeFunction1: TExpTrendFunction;
    Series2: TFastLineSeries;
    Series3: TLineSeries;
    TeeFunction2: TTrendFunction;
    procedure FormCreate(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

implementation

{$IFNDEF CLX}
{$R *.DFM}
{$ELSE}
{$R *.xfm}
{$ENDIF}

procedure TExponentialTrendForm.FormCreate(Sender: TObject);
begin
  inherited;
  Series1.FillSampleValues(1000);
end;

initialization
  RegisterClass(TExponentialTrendForm);
end.

File: TeeNew/Builder2006/Exponential_Trend.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Exponential_Trend.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "Base"
#pragma link "CurvFitt"
#pragma resource "*.dfm"
TExponentialTrendForm *ExponentialTrendForm;
//---------------------------------------------------------------------------
__fastcall TExponentialTrendForm::TExponentialTrendForm(TComponent* Owner)
        : TBaseForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TExponentialTrendForm::FormCreate(TObject *Sender)
{
  Series1->FillSampleValues(1000);
}
//---------------------------------------------------------------------------

